.github/workflows/release-changelog.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           token: ${{ secrets.CHE_BOT_GITHUB_TOKEN }} 
       - name: Create GH Release
         id: create_release
-        uses: ncipollo/release-action@v1.8.0
+        uses: ncipollo/release-action@v1.11.0
```

Comment:
might be worth confirming with @l0rd if he wants to just remove this whole workflow, since releases and changelogs are now done by hand (since about 7.36). I don't know if he uses the output of this as input to his manually created release notes, or if it's just superfluous trash. 

https://github.com/eclipse/che/releases?page=3

Reply:
The release notes are done by hand. We don't use the output of this gh action.